src/world.rs

```diff
@@ -305,6 +305,17 @@ impl World {
         })
     }
 
+    /// Given the ID of an entity without a generation, try to find the matching generation.
+    ///
+    /// This function is useful if you know there's an element with a given index, but you
+    /// don't know its generation.
+    ///
+    /// # Safety
+    /// `id` must correspond to a currently live `Entity`. A despawned or never-allocated `id` will produce undefined behavior.
+    pub unsafe fn entity_from_id(&self, id: u32) -> Entity {
```

Comment:
```suggestion
    /// Given an id obtained from `Entity::id`, reconstruct the still-live `Entity`.
    ///
    /// This function is useful if you know there's an element with a given index, but you
    /// don't know its generation.
    ///
    /// # Safety
    /// `id` must correspond to a currently live `Entity`. A despawned or never-allocated `id` will produce undefined behavior.
    pub unsafe fn find_entity_from_id(&self, id: u32) -> Entity {
```
Whoops, should've noticed the comment too. Also tweaking the name a little more to distinguish it from `fn entity`.